tests/auto/resources/CMakeLists.txt

```diff
@@ -27,4 +27,5 @@ endmacro()
 add_resources_tests(
   test_sugarsession
   test_sugarmockprotocol
+  test_sugarjob
```

Comment:
The test is in fact testing LoginJob. We'll have different tests for other sugarjob subclasses I guess, so better rename this to test_loginjob ?